src/webgpu/shader/execution/memory_model/texture_intra_invocation_coherence.spec.ts

```diff
@@ -0,0 +1,331 @@
+export const description = `
+Test that read/write storage textures are coherent within an invocation.
+
```

Comment:
This could really use the coment from the PR:


> Each invocation is assigned several random writing indices and a single read index from among those
Writes are randomly predicated (except the one corresponding to the read)
Checks that an invocation can read data it has written to the texture previously
Does not test coherence between invocations